slacker/commands/emoji_list_command.py

```diff
@@ -0,0 +1,90 @@
+import os
+import json
+import requests
+
+from argparse import ArgumentParser
+
+from .command import Command
+from slacker.slack_api import SlackAPI
+
+class EmojiListCommand(Command):
+  def name(self):
+    return 'emoji.list'
+
+  def description(self):
+    return 'Lists custom emojis in workspace'
+
+  def aliases(self):
+    return ['emojis']
+
+  def make_parser(self):
+    parser = ArgumentParser(prog = self.name(), description = self.description())
+    parser.add_argument('-p', '--path', type = str,
+                        help='Local file path where to save emojis to')
+    parser.add_argument('-q', '--quiet', action='store_true',
+                        help="Don't print response from Slack API")
+    return parser
+
+  def print_emojis(self, emojis):
+    total = 0
+    for emoji in emojis:
+      print('{:<25} {:<50}'.format(emoji, emojis[emoji]))
+      total += 1
+
+    print('{} custom emojis'.format(total))
+
+  def __download_emojis(self, emojis):
+    save_data = {}
+
+    for emoji in emojis:
+      emoji_url = emojis[emoji]
+      if emoji_url[:5] == 'alias':
+        save_data[emoji] = emoji_url
+        continue
+
+      saved_file_path = self.__download_emoji(emoji, emojis[emoji])
+      save_data[emoji] = saved_file_path
```

Comment:
A nice-to-have for later would be a concurrent download of all emojis to speed things up.

Reply:
I agree. I will add a task for this

Reply:
https://github.com/netromdk/slacker/issues/53